src/components/MenuBar.js

```diff
@@ -367,7 +367,7 @@ class MenuBar extends React.Component {
 
             <Button
               className="createPledge darkBackground"
-              href={`/pledges/new?name=${collective.slug}`}
```

Comment:
Does that trigger a page reload?

Reply:
Good catch, I'll fix that. :+1:

Reply:
Actually, I don't think so because `Button` uses `Router.pushRoute`. I'll do a separate PR to support the cleaner route syntax for the Button href.

Reply:
One annoying thing with `pushRoute` is that it's not scrolling to the top automatically. 

While using the `<Link>` component will do that automatically.

Reply:
@znarf I'm going to merge this now since it's a hidden feature. I'll have a follow up PR to the `Button` component to support scrolling to the top.